src/data/faq_de.json

```diff
@@ -107,7 +107,7 @@
                         "anchor": "rat_how_get_result",
                         "active": true,
                         "textblock": [
-                            "Über den Bereich 'Test registrieren' auf dem Startbildschirm Ihrer Corona-Warn-App, können Sie einen Test hinzufügen, indem Sie den QR-Code scannen. Um das Ergebnis möglichst zeitnah in der App zu erhalten, können Sie den Test in der App aktualisieren, bis das Ergebnis vorliegt. Zwar ruft die Corona-Warn-App Testergebnisse auch automatisch im Hintergrund ab, je nach Betriebssystem kann das aber etwas länger dauern."
+                            "Über den Bereich 'Test registrieren' auf dem Startbildschirm Ihrer Corona-Warn-App, können Sie einen Test hinzufügen, indem Sie den Schnelltest-QR-Code scannen. Um das Ergebnis möglichst zeitnah in der App zu erhalten, können Sie den Test in der App aktualisieren, bis das Ergebnis vorliegt. Zwar ruft die Corona-Warn-App Testergebnisse auch automatisch im Hintergrund ab, je nach Betriebssystem kann das aber etwas länger dauern."

```

Comment:
Um einen Test zu registrieren, können doch sowohl PCR-Test-QR-Codes als auch Schnelltest-QR-Codes über den Bereich "Test registrieren" gescannt werden.

Reply:
This entry is specifically about rapid tests, so I included the more detailed wording.